api/server/user/util.go

```diff
@@ -279,10 +279,18 @@ func toRPCJob(job ffs.StorageJob) (*userPb.StorageJob, error) {
 	}, nil
 }
 
+func fromProtoCid(c string) (cid.Cid, error) {
+	res, err := util.CidFromString(c)
+	if err != nil {
+		return cid.Undef, err
+	}
+	return res, nil
+}
```

Comment:
Isn't this the same of directly using `util.CidFromString`?

Reply:
Yep! Deleting this.